src/Service/SynchronizationService.php

```diff
@@ -161,6 +388,19 @@ public function debounceLiveSync(string $shopContentName)
         return false;
```

Comment:
Does this mean this method is still disabled (no live-sync for now)?

Reply:
Yes, at this time, the live-sync is stay disabled.